README.md

```diff
@@ -44,6 +44,28 @@ but unnamed temporary credentials can be created regardless of your scopes.## AP
 The REST API methods are documented on
 [http://docs.taskcluster.net/](http://docs.taskcluster.net/)
 
+## Sync vs Async
+
+The objects under `taskcluster` (e.g., `taskcluster.Queue`) are python2-compatible and operate synchronously. This allows for taskcluster api access, though for bulk operations will be considerably slower... more than 10x slower in cases where you need to run many api calls that would benefit from concurrency.
```

Comment:
Rather than the sync api is slower inherently, it's more the case that the async api can easily run with concurrency.  As it's written, it almost sounds like a single sync call is up to 10x slower than a single async call.  Let's frame it more like "This allows for making concurrent API calls, which can considerably improve performance for applicable workloads"